docs/compile.md

```diff
@@ -42,6 +42,28 @@ By default, wolfBoot is compiled for ARM Cortex-M3/4/7. To compile for Cortex-M0
 
 `CORTEX_M0=1`
 
+### Speed vs. size
+
+On a number of targets, algorithm may be optimized automatically to use assembly
```

Comment:
"On a number of targets, algorithm may be optimized automatically to use assembly
 optimizations." -> "Some targets support assembly optiomizations by default."?